src/box/applier.cc

```diff
@@ -323,21 +323,14 @@ apply_nop(struct xrow_header *row)
 	return process_nop(&request);
```

Comment:
I don't see new tests in the patch, only old tests fixed. Did you forget to attach its file?

Reply:
Just added one. PTAL.

Reply:
I couldn't understand right away most of them, but I hope it is all fine. LGTM!